app/services/concerns/xccdf/rule_group_rules.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving RuleGroupRules
+  module RuleGroupRules
+    extend ActiveSupport::Concern
+
+    included do
+      def save_rule_group_rules
+        @op_rules_and_rule_groups = @op_rule_groups + @op_rules
+        @rule_group_rules = rgr_with_requires + rgr_with_conflicts
+        @rule_group_rules = @rule_group_rules.compact
+
+        ::RuleGroupRule.import!(
+          @rule_group_rules.select(&:new_record?), ignore: true
+        )
+
+        rule_base = ::RuleGroupRule.joins(included_rule: :benchmark)
+        rule_group_base = ::RuleGroupRule.joins(included_rule_group: :benchmark)
+
+        rgr_links_to_remove(rule_base).delete_all
+        rgr_links_to_remove(rule_group_base).delete_all
```

Comment:
If you prefix these, please prefix the existing one in `Xccdf::ProfileRules` for semantical correctness.